### PR TITLE
pl translation

### PR DESCRIPTION
**Pull request by gienkov**

### CODE REVIEW

**Comment by gienkov** [comment]
Ready for 3.4.0

**Comment by SilentSpike** [comment]
:+1:

File: addons/map_gestures/stringtable.xml [hunk 1]
```diff
@@ -4,116 +4,179 @@
...
+            <Polish>Maks. zasięg gestów</Polish>
             <Russian>Макс. дистанция жестов на карте</Russian>
         </Key>
         <Key ID="STR_ACE_map_gestures_maxRange_description">
             <English>Max range between players to show the map gesture indicator [default: 7 meters]</English>
             <Portuguese>Distância max. entre os jogadores para mostrar o indicador de gesto no mapa [padrão: 7 metros]</Portuguese>
+            <Polish>Maksymalny zasięg, w obrębie którego gesty będą widoczne dla graczy [domyślnie: 7 metrów]</Polish>
             <Russian>Макс. дистанция между игроками для отображения жестов на карте [по-умолчанию: 7 метров]</Russian>
         </Key>
+        <Key ID="STR_ACE_map_gestures_defaultLeadAlpha_displayName">
```
Comment (PabstMirror) [inline addons/map_gestures/stringtable.xml]: We don't need these 8 "alpha" entries anymore.  But we can just remove after merge.

Comment (gienkov) [inline addons/map_gestures/stringtable.xml]: Sorry I didn't know that. I dis this PR back then and updated master today.
I can remove this on my side if needed?

**Comment by gienkov** [comment]
Removed 'alpha' entries.

File: addons/refuel/stringtable.xml [hunk 2]
```diff
@@ -1,129 +1,160 @@
...
             <German>Stecke Zapfpistole zurück ...</German>
+            <Polish>Zwracanie nalewaka ...</Polish>
         </Key>
         <Key ID="STR_ACE_Refuel_CheckFuelCounter">
             <English>Check fuel counter</English>
             <German>Tankuhr ansehen</German>
+            <Polish>Sprawdź wskaźnik paliwa</Polish>
         </Key>
         <Key ID="STR_ACE_Refuel_CheckFuelCounterAction">
-            <English>>Checking fuel counter ...</English>
```
Comment (gienkov) [inline addons/refuel/stringtable.xml]: I've fixed typo here

File: addons/rearm/stringtable.xml [hunk 3]
```diff
@@ -1,72 +1,89 @@
...
+﻿<?xml version="1.0" encoding="utf-8"?>
 <Project name="ACE">
     <Package name="Rearm">
         <Key ID="STR_ACE_Rearm_RearmSettings_Module_DisplayName">
             <English>Rearm Settings</English>
             <German>Aufmunitioniereinstellungen</German>
+            <Polish>Ustawienia przezbrajania</Polish>
         </Key>
         <Key ID="STR_ACE_Rearm_RearmSettings_Module_Description">
-            <English>This module allows you to tweak repair system settings.</English>
```
Comment (gienkov) [inline addons/rearm/stringtable.xml]: Fixed wrong name here (rearm, not repair)

**Comment by jokoho48** [comment]
`addons/repair/stringtable.xml` 1,013 additions, 1,013 deletions
that sound not good

**Comment by gienkov** [comment]
How to fix that?

**Comment by SilentSpike** [comment]
Possibly an EOL difference?

Can you try putting `file addons/repair/stringtable.xml` into your git terminal and checking what it says?
The version is master has:
>XML 1.0 document, UTF-8 Unicode (with BOM) text, with CRLF line terminators

**Comment by jonpas** [comment]
It's probably saying that due to the merge of master branch, diff looks fine.

**Comment by PabstMirror** [comment]
:+1: Doskonale

**Comment by gienkov** [comment]
I've fixed that :)

**Pull request merged**